src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -2452,6 +2465,23 @@ public boolean onSolderingToolRightClick(
         return true;
     }
 
+    public boolean mUseMultiparallelMode = false;
+
+    @Override
+    public boolean onWireCutterRightClick(
```

Comment:
Wire cutter will collide with mixer, ABS and QFT.

Reply:
fixed for mixer and ABS, QFT batchmode has to be done separately, will make another pr and fix the wire cutter issue there